com.unity.render-pipelines.universal/Documentation~/complex-lit-shader.md

```diff
@@ -0,0 +1,90 @@
+# Complex Lit Shader
+
+The Complex Lit Shader is similar to the Lit shader, but it adds advanced material features. These features often are considerably more resource intensive and therefore are only supported on relatively new hardware. Complex Lit shader is always rendered using forward rendering and used as a fallback shader in deferred rendering. Complex Lit contains all the features that are incompatible with deferred rendering. This Shader uses the most computationally heavy [shading model](shading-model.md) in the Universal Render Pipeline (URP). 
```

Comment:
`relatively new hardware` is not very descriptive, should we say supported on compute or requires SM 4.5?

Reply:
fixed